reactioncommerce/reaction
# Enable anonymous users to checkout with Stripe Connect

Resolves #2832 

### How to test
- As marketplace admin set Stripe Connect as payment method
- Load Reaction in a new incognito window. Make sure this is a new incognito session
- Place an item in the cart, and proceed to checkout
- Click on `Continue As Guest` button
- Enter an email address in the form
- Continue with checkout steps. Observe that the checkout process is completed and you get the `orders/completed` page.
